keyboards/keyboardio/atreus/config.h

```diff
@@ -0,0 +1,82 @@
+/* Copyright (C) 2019, 2020  Keyboard.io, Inc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+
+#define VENDOR_ID       0x1209
```

Comment:
I'm a bit concerned about this VID.
It's already used by a few boards with VIA support:
https://github.com/the-via/keyboards/blob/master/src/duckle/vitamins_included.json#L3
https://github.com/the-via/keyboards/blob/master/src/jiran/jiran.json#L3

(The fact that 2 VIA-enabled board already uses it is another topic.)

Reply:
1209 is the VID for pid.codes -- it's properly registered: http://pid.codes/1209/2303/